### ISSUE TITLE
Updated Logos

### ISSUE THREAD

**Issue by Pha6se** [pizzaboxer/bloxstrap#118]
I've slightly updated the Bloxstrap logos to better fit with the Roblox refresh of 2022, with a bigger gap in the center. Whether you use them or not is completely up to you!

Maybe it could be an optional setting?

![bloxstrap_white_128x128](https://user-images.githubusercontent.com/101849117/227785966-6cd39e2b-bd3c-4abb-b086-8b3e808ae43d.png)
![bloxstrap_hanpurple_128x128](https://user-images.githubusercontent.com/101849117/227785967-e871b91e-6318-4d08-8128-f8d40a7cc7c7.png)
![bloxstrap_metallic_128x128](https://user-images.githubusercontent.com/101849117/227785968-334f0c96-d0d8-4a92-9ec5-6ad536ab60de.png)
![bloxstrap_red_128x128](https://user-images.githubusercontent.com/101849117/227785969-01f632e9-8a91-4769-bff5-8d6a7c72633c.png)
![bloxstrap_black_128x128](https://user-images.githubusercontent.com/101849117/227785970-73d83656-d7d3-4d53-b6cd-d098fda10a27.png)

**Comment by pizzaboxer** [comment]
Uhhhh... does the size of the center gap not match?
![image](https://user-images.githubusercontent.com/41478239/227788394-0eeae2c7-640a-4c31-a542-4ca879a1f78c.png)

**Comment by Pha6se** [comment]
Yes.. I realized that the gap was widened on the Bloxstrap logo too. However, it’s still ever-slightly smaller than the refreshed icon. I feel like widening the gap a little makes it look like the refreshed one, especially at a distance or when it’s shrunk down. For example, in the taskbar

**Comment by pizzaboxer** [comment]
Huh, I guess I'll have to redo that it at some point. I redid it back in October for the new logo update, I guess I didn't really check it properly. Might also take the time to figure out how to apply a shiny gradient effect this time.
